### PR TITLE
Update greedy-best-first-search.md

### LINKED ISSUE #3565: [Edit] Subject: Greedy Best First Search Algorithm
### Type of Edit (select all that apply)

Clarify unclear content and/or terms

### Description (optional)

Greedy Best First Search is not properly explained.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [X ] Issue is assigned

### PR DESCRIPTION
**Pull request by Sahilkumar19**
Fixes #3565 

Description
This PR updating and adding new feature in the Greedy Best First Search algorithm which are the important one and every folks out there should know them.
closes #3565 

Type of Change
-Adding a new entry
-Editing an existing entry (fixing a typo, bug, issues, etc)
-Updating the documentation

Checklist
- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/ai/concepts/search-algorithms/terms/greedy-best-first-search/greedy-best-first-search.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ CatalogContent:
...
-**Greedy best-first search** is an informed search algorithm where the evaluation function is strictly equal to the heuristic function, disregarding the edge weights in a weighted graph. To get from a start node to a target node, the lowest value resulting from some heuristic function, **h(x)**, is considered as the successive node to traverse to. The goal is to choose the quickest and shortest path to the target node.
+**Greedy best-first search** is an informed search algorithm where the evaluation function is strictly equal to the heuristic function, disregarding the edge weights in a weighted graph(because we care about the heuristic value). In ordre to search for a goal node expand the node that is closest to the goal as determined by the heuristic function.Assume that is likely to lead a solution quickly.The solution from Greedy Best First Search may not be optimal since a shorter path may exist.In this algorithm search cost is minimum since solution found without expanding a node that is not on the solution path.This algorithm is minimal but not complete since it can lead to a dead end.It's called "Greedy" since at each step it tries to get as close to the goal as it can.
```
Comment (yangc95) [inline content/ai/concepts/search-algorithms/terms/greedy-best-first-search/greedy-best-first-search.md:15]: Can we keep the original wording but expand on certain terms like "heuristic function" in a concise manner? The change is quite wordy and doesn't necessarily present any new information.

Comment (yangc95) [inline content/ai/concepts/search-algorithms/terms/greedy-best-first-search/greedy-best-first-search.md:15]:
```suggestion
**Greedy best-first search** is an informed search algorithm where the evaluation function is strictly based on the heuristic function, regardless of edge weights in a weighted graph. It selects the next node to traverse based on the lowest value obtained from a heuristic function, denoted as _h(x)_. The primary objective is to find the quickest path to the target node.
```

**Review by yangc95 — CHANGES_REQUESTED** [review]
@Sahilkumar19 I appreciate your input in refining the text. It's great to collaborate and improve the content. However, to maintain clarity and conciseness, we should aim for meaningful edits.

Feel free to continue making suggestions, and let's work together to enhance the text while keeping it clear and informative.

**Comment by Sahilkumar19** [comment]
> @Sahilkumar19 I appreciate your input in refining the text. It's great to collaborate and improve the content. However, to maintain clarity and conciseness, we should aim for meaningful edits.
> 
> Feel free to continue making suggestions, and let's work together to enhance the text while keeping it clear and informative.

Right, thanks for pointing it out that we should aim for meaningful edits.

**Comment by Sahilkumar19** [comment]
Can I close this PR? as I have already made one PR for the same issue.

**Comment by yangc95** [comment]
> Can I close this PR? as I have already made one PR for the same issue.

Sure! I can go ahead and close this PR.

**Pull request closed without merging**